### ISSUE TITLE
Drawing stroke is inaccurate at high scale

### ISSUE THREAD

**Issue by Frojdholm** [nannou-org/nannou#799]
Using `nannou` version 0.17.1 when drawing using a high `scale` and `no_fill()` the strokes are not accurate. I'm on Windows 10 using `rustc 1.55.0 (c8dfcfe04 2021-09-06)`.

See the following example

```rust
use nannou::prelude::*;

fn main() {
    nannou::sketch(view).run();
}

fn view(app: &App, frame: Frame) {
    // Inaccurate stroke
    let draw = app.draw().scale(800.0);
    draw.rect().w_h(0.03, 0.03).stroke_color(SNOW).no_fill().stroke_weight(1.0 / 800.0);

    // Fill works
    // let draw = app.draw().scale(800.0);
    // draw.rect().w_h(0.03, 0.03).color(SNOW);

    // Not using scale works
    // let draw = app.draw();
    // draw.rect().w_h(0.03 * 800.0, 0.03 * 800.0).stroke_color(SNOW).no_fill().stroke_weight(1.0);

    draw.to_frame(app, &frame).unwrap();
}
```
Which produces this image:
![inaccurate_stroke](https://user-images.githubusercontent.com/3251566/135892006-e8e1d76c-edd8-4a58-8581-9e04fe40c4bc.png)

**Comment by Frojdholm** [comment]
After some investigation it looks like it's either a bug or threshold in Lyon causing vertices to be merged when tessellating the stroke with small stroke width.

Since the transform is applied after tessellating the points and stroke width passed to Lyon are very small.

**Comment by Frojdholm** [comment]
A fix is now merged in Lyon and available from 0.17.10.

**Comment by mitchmindtree** [comment]
Closed via #801, thanks again @Frojdholm!
